wolfcrypt/src/ge_operations.c

```diff
@@ -9192,25 +9192,27 @@ void ge_scalarmult_base(ge_p3 *h,const unsigned char *a)
 }
 
 
+#define SLIDE_SIZE 256
+
 /* ge double scalar mult */
 static void slide(signed char *r,const unsigned char *a)
```

Comment:
This function has weird whitespacing and alignment.

Reply:
incumbent code (d89d524570), not worth disrupting.